econml/tests/test_ortho_learner.py

```diff
@@ -149,7 +149,7 @@ def score(self, Y, T, W=None, nuisances=None):
         sigma = 0.1
         y = X[:, 0] + X[:, 1] + np.random.normal(0, sigma, size=(10000,))
         est = _OrthoLearner(ModelNuisance(LinearRegression(), LinearRegression()), ModelFinal(),
-                            n_splits=2, discrete_treatment=False, random_state=None)
+                            n_splits=2, discrete_treatment=False, discrete_instrument=False, random_state=None)

```

Comment:
For `_OrthoLearner` itself there are no defaults (and never have been for any of the exisitng arguments).

I made a slight change to the definition there to force all of `discrete_treatment`, `discrete_instrument`, `n_splits`, and `random_state` to be keyword only, so that you not only need to specify them (because there are no defaults) but you need to specify them by name; I did this because I was afraid that if we add any other arguments at some point then there's a chance that we might make a mistake when inserting it into the positional argument list in the wrong place.